integration-tests/oidc-code-flow/src/test/java/io/quarkus/it/keycloak/CodeFlowTest.java

```diff
@@ -111,6 +118,67 @@ public void testTokenTimeoutLogout() throws IOException, InterruptedException {
         }
     }
 
+    @Test
+    public void testRPInitiatedLogout() throws IOException, InterruptedException {
+        Keycloak keycloak = KeycloakRealmResourceManager.createKeycloakClient();
```

Comment:
That should be in `KeycloakRealmResourceManager` ?

Reply:
I'm not sure if it is worth doing that as we are updating the realm itself. Realm-level settings. I think tests should be able to change the configuration at Keycloak at their will, like this one. Especially when you have some more dynamic test scenario that depends on changing config.

Reply:
It is not a big deal but it just complicates the actual test a lot, simply creating all what is needed in a dedicated class makes things cleaner a lot.

Reply:
OK. Going to update.